api/main.py

```diff
@@ -96,8 +97,25 @@ async def post_node(node: Node, token: str = Depends(get_user)):
             obj = await db.create(node)
```

Comment:
I still think we can just remove the whole `if` / `else` logic here and just call `db.create()`.  If there is a `node.id`, an exception will be raised by `db.create()` and then the exception handler below will return HTTP 400 which seems like what would be expected with invalid data.

Reply:
Done the changes.